api/initializer.go

```diff
@@ -0,0 +1,49 @@
+package api
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"net/http"
+
+	_ "github.com/lib/pq"
+)
+
+const (
+	host     = "db"
+	port     = 5432
+	user     = "admin"
+	password = "admin"
+	dbname   = "postgres"
+)
+
+const PORT = 8080
+
+var db *sql.DB
```

Comment:
It's recommended to separate the database layer from the API layer. This can be achieved by splitting them into different files, but what's crucial is to maintain the appropriate dependency between these two layers.

Ensure that you initialize your database first and then pass the connection to the API. This approach promotes better organization and separation of concerns.